testing/tests/DevExpress.ui.widgets/buttonGroup.tests.js

```diff
@@ -318,6 +318,263 @@ QUnit.module('Events', () => {
     });
 });
 
+QUnit.module('selectionMode', () => {
```

Comment:
Pls, check the buttonGroup.selection.tests file and transfer your scenarios to that file

Reply:
Yep, fixed.
Just want to note, that new tests uses API (selectedItemKeys and selectedItems) for storing selected items.
Not clicks